scanner/src/main/kotlin/scanners/scancode/ScanCode.kt

```diff
@@ -200,7 +200,8 @@ class ScanCode(
         }
 
         val result = getRawResult(resultsFile)
-        val summary = generateSummary(startTime, endTime, path, result)
+        val parseLicenseExpressions = scanCodeConfiguration["parseLicenseExpressions"]?.toBoolean() == true
+        val summary = generateSummary(startTime, endTime, path, result, parseLicenseExpressions)
 
```

Comment:
I think it is a bit confusing that `parseLicenseExpressions` is by default `false` while the last argument in `generateSummary` is by default `true`. Is this necessary for the tests to run?

Reply:
It's not strictly necessary but convenient, see [above](https://github.com/oss-review-toolkit/ort/pull/3564#discussion_r573731075).